enterprise/frontend/src/embedding-sdk/lib/theme/default-component-theme.ts

```diff
@@ -138,5 +138,62 @@ export function getEmbeddingComponentOverrides(
         ...(theme?.popover?.zIndex && { zIndex: theme.popover.zIndex }),
       },
     },
+    // TODO: fix those, they're not all correct
+    ModalRoot: {
+      defaultProps: {
+        withinPortal: true,
```

Comment:
I don't think this is going to work in all cases, there are some cases that a floating element needs to not be in the portal, and a case that it needs to be in the modal. This might break some of the default behavior for modal/popover/tooltip in some cases.

I'm just worried we're digging ourselves a hole with this. Since the combinations of floating elements are quite endless.

What I did for the main app is to inject styles directly into these floating element roots.

For example, this is for setting tooltip styles.
https://github.com/metabase/metabase/blob/7bd720439edd6254b13f1598a0f5bc689c73f9cb/frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.module.css#L33

That's one more option in case you want to try it.